.gitignore

```diff
@@ -7,7 +7,7 @@
 .svn
 *~
 manifest.appcache
-
+/.idea
```

Comment:
@mwargers .idea are configuration files for the Jetbrains products. So just like .project is in eclipse, .idea is in Pycharm or any other Jetbrains product.